src/components/Navbar/index.js

```diff
@@ -39,9 +39,11 @@ export default ({ pathname }) => {
   }
 
   return (
-    <div ref={ref} className='header'>
-      {searchNode && ReactDOM.createPortal(<Search />, searchNode)}
-      {notificationsNode && ReactDOM.createPortal(<NotificationsFeed />, notificationsNode)}
+    <div style={{zIndex: 25}}>
+      <div ref={ref} className='header'>
+        {searchNode && ReactDOM.createPortal(<Search />, searchNode)}
+        {notificationsNode && ReactDOM.createPortal(<NotificationsFeed />, notificationsNode)}
+      </div>
```

Comment:
This creates an overlap issue with global navigation. Instead change the [`top`](https://github.com/santiment/app/blob/master/src/ducks/Watchlists/Widgets/TopBar/TopBar.module.scss#L60) style rule to match the header height.
![CleanShot 2022-05-24 at 12 42 57@2x](https://user-images.githubusercontent.com/25135650/170002387-e5dc7038-beb5-47db-b75b-81b576d08233.jpg)